i18n/en.json

```diff
@@ -4614,6 +4614,10 @@
     "id": "model.config.is_valid.sql_max_conn.app_error",
     "translation": "Invalid maximum open connection for SQL settings.  Must be a positive number."
   },
+  {
+    "id": "model.config.is_valid.sql_max_lifetime_conn.app_error",
+    "translation": "Invalid maximum lifetime connection for SQL settings.  Must be a positive number."
```

Comment:
See comments above and below, but this will probably become "Invalid connection maximum lifetime for SQL settings. Must be a non-negative number."

Also, looks like have some redundant spaces between sentences in the various translations here. I'm 0/5, but I suspect they were a mistake and yours can eliminate same.